lib/manager/helm-requirements/extract.ts

```diff
@@ -0,0 +1,61 @@
+import is from '@sindresorhus/is';
+import upath from 'upath';
+import yaml from 'js-yaml';
+
+import { logger } from '../../logger';
+import { PackageFile, PackageDependency } from '../common';
+
+export async function extractPackageFile(
+  content: string,
+  fileName: string
+): Promise<PackageFile> {
+  try {
+    const baseDir = upath.parse(fileName).dir;
+    const chartFileName = upath.join(baseDir, 'Chart.yaml');
+    const chartContents = await platform.getFile(chartFileName);
```

Comment:
`platform` is no longer a global. You need `import { platform } from '../../platform';`